### PR TITLE
Russian towel mode with button

### PR DESCRIPTION
**Pull request by C-Pro**

### CODE REVIEW

File: bot/skills/towel_mode.py [hunk 1]
```diff
@@ -78,18 +83,31 @@ def check_for_reply(update: Update, context: CallbackContext):
...
+def butt_press(update: Update, context: CallbackContext):
+    quarantine, quarantine_min = get_quarantine_and_quarantine_min(context)
+    minute = quarantine.get(update.effective_user.id)
+    if minute is None:
+        return
+    chat_id = update.effective_chat.id
+    user = update.effective_user
+    query = update.callback_query
+    logger.info(query.data)
+    if query.data == 42 or True:
```
Comment (egregors) [inline bot/skills/towel_mode.py]: @C-Pro looks like this condition is always True

File: bot/skills/towel_mode.py [hunk 2]
```diff
@@ -60,14 +63,16 @@ def quarantine_user(user: User, chat_id: str, context: CallbackContext) -> None:
     minute = datetime.datetime.utcnow().minute
     quarantine[user.id] = minute
     quarantine_min[minute].append(user.id)
+
+    markup = InlineKeyboardMarkup([
+        [InlineKeyboardButton("Я не бот!", callback_data=42)]])
```
Comment (egregors) [inline bot/skills/towel_mode.py]: @C-Pro nice magic number! 🎱🎱🎱

File: bot/skills/towel_mode.py [hunk 3]
```diff
@@ -78,18 +83,31 @@ def check_for_reply(update: Update, context: CallbackContext):
...
+@run_async
+def butt_press(update: Update, context: CallbackContext):
+    quarantine, quarantine_min = get_quarantine_and_quarantine_min(context)
+    minute = quarantine.get(update.effective_user.id)
+    if minute is None:
+        return
+    chat_id = update.effective_chat.id
+    user = update.effective_user
+    query = update.callback_query
+    logger.info(query.data)
```
Comment (egregors) [inline bot/skills/towel_mode.py]: @C-Pro need context for log message, I guess

File: bot/skills/towel_mode.py (hunk 1 above)
Comment (C-Pro) [reply bot/skills/towel_mode.py]: yep. log shows that 42 is passed here, but condition fails. IDK maybe it comes as a string. Explicitly converted it to string

File: bot/skills/towel_mode.py (hunk 2 above)
Comment (C-Pro) [reply bot/skills/towel_mode.py]: For your total satisfaction I moved it to properly-named constant

File: bot/skills/towel_mode.py (hunk 3 above)
Comment (C-Pro) [reply bot/skills/towel_mode.py]: just removed it

**Pull request merged**